### PR TITLE
feat: stabilize NEP 492

### PR DESCRIPTION
**Pull request by bowenwang1996**
Stabilize https://github.com/near/NEPs/pull/492. Restrict the creation of non-implicit top-level account that are longer than 32 bytes. Only the registrar account can create them. More context on the proposal:

- The NEP has been officially approved in September (https://github.com/near/NEPs/pull/492#issuecomment-1703537744).
- #9589 implements the feature and includes tests checking that TLA can no longer be created in the new protocol version.

### CODE REVIEW

**Comment by bowenwang1996** [comment]
@akhi3030 do you know why I couldn't request review from @jakmeier ? Is it because of the github organization setup?

**Comment by akhi3030** [comment]
> @akhi3030 do you know why I couldn't request review from @jakmeier ? Is it because of the github organization setup?

Hmm, it is weird indeed.  I'll read some github docs to figure it out.

File: CHANGELOG.md
Comment (jakmeier) [inline CHANGELOG.md:7]: Hm, I think linking to #9589 would be more helpful for people reviewing changes. Or maybe just link to both?

**Review by jakmeier — APPROVED** [review]
LGTM. Left one comment regarding the changelog entry.


As a general note, when stabilizing a protocol feature, the PR itself is usually very simple but it is nevertheless a crucial change. Therefore, I love it when the PR author describes what testing has ben done and why it's ready to be stabilized. It usually only takes 30s to write it in the commit message but potentially saves the reviewers much more time.
Of course, in this case, for me, I already have all the context. But adding this information makes it more accessible for anybody else.

For example:

- The NEP has been officially approved in September (https://github.com/near/NEPs/pull/492#issuecomment-1703537744).
- #9589 implements the feature and includes tests checking that TLA can no longer be created in the new protocol version.

**Review by akhi3030 — APPROVED** [review]
Approving to unblock.

**Comment by akhi3030** [comment]
> @akhi3030 do you know why I couldn't request review from @jakmeier ? Is it because of the github organization setup?

Looking at https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review, I see the following:

"""
Organization members with write access can also assign a pull request review to any person or team with read access to a repository.
"""

So the simple answer is that @ jakmeier does not have read access to the repo.  Not sure if github allows giving the world read access.  It does not appear that this is possible.  https://github.com/near/nearcore/settings/access?query= shows who has what kind of access to the repo currently.  I will start an internal thread to discuss making changes here.

**Comment by jakmeier** [comment]
For what it's worth, tagging me in a comment will do the same for me as adding me as reviewer. Both will show up as GH notification and I will take a look.

**Pull request merged**
